README.md

```diff
@@ -91,12 +77,59 @@ updates automatically.
 
 4.  Click on the <img src="ext/images/play-audio.svg" width="16" height="16" alt="loudspeaker icon"> _speaker_ button to hear the term pronounced by a native speaker. If an audio sample is not available, you will hear a short click instead. For more options, see [Audio Configuration](./docs/audio.md).
 
-5.  Click on individual kanji in the term definition results to view additional information about those characters, including stroke order diagrams, readings, meanings, as well as other useful data.
+5.  Click on individual kanji in the term definition results to view additional information about those characters, including stroke order diagrams, readings, meanings, and other useful data.
 
     <img src="resources/images/search-popup-kanji.png" width="300" alt="popup with kanji details">
 
 To further enhance your Yomitan experience, it's worth [integrating with Anki](./docs/anki-integration.md), a spaced-repetition flashcard program to help solidify the words you encounter.
 
+## Changes
+
+> While we have made some substantial changes, the majority of the extension's functionality is thanks to hard work of [FooSoft](https://foosoft.net/) and many other incredible open source contributors from 2016-2023.
+
+Major changes:
+
+- 💪 Completed the Manifest V2 → V3 transition, [read why here](https://developer.chrome.com/blog/resuming-the-transition-to-mv3/)!
+- ⚡ Switched to using ECMAScript modules and npm-sourced dependencies.
+- 🔧 Implemented an end-to-end CI/CD pipeline.
+- 🧪 Switched to standard testing frameworks, vitest and playwrights.
+
+In addition, we have made important bug fixes and minor enhancements:
```

Comment:
Should mention supporting other languages here I think.

Reply:
done